plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/table/actions/install.tsx

```diff
@@ -51,6 +51,44 @@ export default function Install( props: InstallProps ) {
 		);
 	};
 
+	const handleInstallError = ( error: any ) => {
+		loadSubscriptions( false ).then( () => {
+			let errorMessage = sprintf(
+				// translators: %s is the product name.
+				__( '%s couldn’t be installed.', 'woocommerce' ),
+				props.subscription.product_name
+			);
+			if ( error?.success === false && error?.data.message ) {
+				errorMessage += ' ' + error.data.message;
+			}
+			addNotice(
+				props.subscription.product_key,
+				errorMessage,
+				NoticeStatus.Error,
+				{
+					actions: [
+						{
+							label: __( 'Try again', 'woocommerce' ),
+							onClick: install,
```

Comment:
I'm getting a linter error, `'install' was used before it was defined.`, so perhaps just switch the order of the functions?

Reply:
f6000649002433a156d014aba76ba5e606d5522c